Testing/ContinuousIntegration/AzurePipelinesMacOSPython.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         export ITK_WRAP_PYTHON=ON
         export CTEST_OUTPUT_ON_FAILURE=1
 
-        ctest -S ITK-dashboard/azure_dashboard.cmake -V -j 4
+        ctest -S ITK-dashboard/azure_dashboard.cmake -V -j 4 -R Python
```

Comment:
We could keep all tests to ensure we get all the Python ones on macOS since it does not have timing pressure.

Reply:
Thanks for the review. I was indeed wondering if I should filter the tests. I think it makes sense to keep it without this flag on MacOS to ensure that if this flag is not used, all the expected tests are run.

Reply:
That being said, if #688 gets merged, then there should be less time pressure anyway.